### PR TITLE
Add ability to specify additional security groups

### PR DESCRIPTION
**Pull request by adamcrews**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -355,3 +355,9 @@ variable "backtrack_window" {
   description = "The target backtrack window, in seconds. Only available for aurora engine currently. Must be between 0 and 259200 (72 hours)"
   default     = 0
 }
+
+variable "vpc_security_group_ids" {
+  type        = list(string)
+  description = "Additional security group ids to apply to the cluster, in addition to the default security group composed from `allowed_cidr_blocks`"
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Additional security group IDs to apply to the cluster, in addition to the provisioned default security group with ingress traffic from existing CIDR blocks and existing security groups
```

**Comment by maximmi** [comment]
@adamcrews please, run `terraform fmt`. Tests failed due to `variables.tf` formatting

**Comment by adamcrews** [comment]
Fixed the `fmt` error, and rebased since there was another PR merged before this one.

**Pull request merged**